src/auxiliary/t8code.jl

```diff
@@ -0,0 +1,452 @@
+"""
+    init_t8code()
+"""
+function init_t8code()
+  t8code_package_id = t8_get_package_id()
+  if t8code_package_id >= 0
+    return nothing
+  end
+
+  # Initialize `t8code` with log level ERROR to prevent a lot of output in AMR simulations
+  t8_init(SC_LP_ERROR)
+
+  return nothing
+end
+
+@inline t8_mpi_comm() = mpi_comm().val
```

Comment:
Is this based on document API of MPI.jl?

Reply:
This routine is deprecated and was deleted. Thanks for pointing this out.